src/utils/string.ts

```diff
@@ -0,0 +1,6 @@
+export function convertToSlug(text: string) {
```

Comment:
What is convert to slug?

Reply:
Can you add a description for the function?

Reply:
any string.    nguyễn hoài danh => nguyen-hoai-danh

Reply:
If you have time, please put a description above the function, before regex is hard to read.

Reply:
got it.